### DISCUSSION TITLE
since the xcodeproj is generated, i can't configure the example project

### DISCUSSION THREAD

**Discussion by QDFish** [callstack/react-native-builder-bob#615] [Q&A] [answered]
for example, I just need to add NFC capability, and anyone who clones my project can test this feature without configuring it again， can I have any method to use an xcodeproj file in the example dir again?

**Comment by atlj** [comment, marked as answer]
> Since the xcodeproj is generated

I think you mean the [React Native Test App](https://github.com/microsoft/react-native-test-app). You can either [configure the test app](https://github.com/microsoft/react-native-test-app/wiki/Manifest-%28app.json%29) or stick with the vanilla test app in this case. If you want to use the vanilla test app, you can create the project again and choose the vanilla option when it asks you what type of test app you want to use.

**Reply by QDFish** [reply]
it help, thanks
